### PR TITLE
Fix bugs in deb build script 

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
The deb script did not extract the version number
generated by libtool correctly.
To fix this issue, the deb script now parses the
version  number from filename of the created .so
library file

### CODE REVIEW

**Comment by andreas-bok-sociomantic** [comment]
ping

**Comment by ben-palmer-sociomantic** [comment]
I would rephrase the commit title `Fix wrong library version for deb packaging` as `Fix library version for deb packaging`. Also the commit message has the misspelling `LIBDRIZLE_LIBRARY_VERSION` and two spaces between `version` and `number from filename of the created .so`. Otherwise LGTM.

**Comment by andreas-bok-sociomantic** [comment]
Updated based on comments from @ben-palmer-sociomantic

**Comment by ben-palmer-sociomantic** [comment]
Do we need the `[ci skip]` that is now in the commit messages?

**Comment by andreas-bok-sociomantic** [comment]
> Do we need the [ci skip] that is now in the commit messages

When it is there it skips the CI testing.

**Comment by ben-palmer-sociomantic** [comment]
> When it is there it skips the CI testing.

Do we need to skip the testing for this PR though?

**Comment by andreas-bok-sociomantic** [comment]
No, sorry was just the force push which didn't have a changes to the script. Will rm the ci skip  :)

**Comment by andreas-bok-sociomantic** [comment]
Fixed ci msgs take 2

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Comment by ben-palmer-sociomantic** [comment]
This pull request has been rebased via `git hub pull rebase`. Original pull request HEAD was 86a525f401816875fef3b0530ba2e41f5d47b279, new (rebased) HEAD is 070ba46567cf05e9d7262bb37f8273aaf813c646
